pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>JVips</groupId>
     <artifactId>JVips</artifactId>
-    <version>1.0.1</version>
+    <version>0.0.0-SNAPSHOT</version>
```

Comment:
not 1.0.1-SNAPSHOT?

Reply:
0.0.0 makes it quite explicit that this is *not* an actual version, rather a placeholder. Version will now be extrapolated from vips version, so I'd prefer to avoid setting a version here.

Reply:
Just tested and `<version>` is required. I'd prefer to leave it as 0.0.0-SNAPSHOT then.

Reply:
We can align the version with the libvips one once we will have all implemented bindings

Reply:
Ok for me for 0.0.0-SNAPSHOT.
For aligning the version with the libvips one, we can perhaps do it in another patch.